test/math/BoolVectorListTest.ooc

```diff
@@ -84,30 +84,30 @@ BoolVectorListTest: class extends Fixture {
 			(list, closed) free()
 		})
 		this add("logical operators", func {
-			list := this _createFromText(t"11010")
-			other := this _createFromText(t"10001")
+			list := this _createFromString("11010")
+			other := this _createFromString("10001")
 			listAndOther := list && other
 			listOrOther := list || other
 			expect(listAndOther tally(true), is equal to(1))
 			expect(listOrOther tally(true), is equal to(4))
 			(list, other, listAndOther, listOrOther) free()
 		})
 		this add("toFloatVectorList", func {
-			list := this _createFromText(t"11001")
+			list := this _createFromString("11001")
 			floatList := list toFloatVectorList()
 			expect(floatList sum, is equal to(3.0f) within(0.01f))
 			(floatList, list) free()
 		})
-		this add("toText", func {
-			list := this _createFromText(t"101")
-			text := list toText() take()
-			expect(text, is equal to(t"true\nfalse\ntrue"))
+		this add("toString", func {
+			list := this _createFromString("101")
+			text := list toString()
+			expect(text, is equal to("true\nfalse\ntrue\n"))
 			(text, list) free()
 		})
 	}
-	_createFromText: func (content: Text) -> BoolVectorList {
+	_createFromString: func (content: String) -> BoolVectorList {
 		result := BoolVectorList new()
-		for (i in 0 .. content count) {
+		for (i in 0 .. content size) {
```

Comment:
Funny story: `String` has a method called `count` which does something completely different from `Text count`. Before I changed this for-loop, `rock` would actually compile `for (i in 0 .. content count)` without any warnings or anything (i.e. the end of the loop was a pointer to a method?) but would of course crash horribly once the program ran.